package.json

```diff
@@ -49,7 +49,7 @@
 		"extends @nextcloud/browserslist-config"
 	],
 	"engines": {
-		"node": "^14.0.0",
-		"npm": "^7.0.0"
+		"node": "^14 || ^16",
```

Comment:
right now everything seems to be targeting 14, so let's revert the node/npm requirements

Reply:
Fixed